guides/runbooks/orca-zombie-executions.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: single
+title:  "Orca: Zombie Executions"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Aliases: `orphaned execution`
+
+A pipeline or task is "stuck" in a `RUNNING` state but not doing anything.
+
+# Diagnosis
+
+Logs will be emitted regularly for Executions that are currently running in Orca.
```

Comment:
an example of how this log line looks like describing pipeline id might be useful here